ESH-INF/thing/_channels.xml

```diff
@@ -3,15 +3,31 @@
 	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
-	<!-- Color Channel -->
-	<channel-type id="color_color">
-		<item-type>Color</item-type>
-		<label>Color</label>
-		<description>The color channel allows to control the color of a light.
+    <!-- Color Channel -->
+    <channel-type id="color_color">
+        <item-type>Color</item-type>
+        <label>Color</label>
+        <description>The color channel allows to control the color of a light.
             It is also possible to dim values and switch the light on and off.
         </description>
-		<category>ColorLight</category>
-	</channel-type>
+        <category>ColorLight</category>
+    </channel-type>
+
+    <!-- Color Channel -->
+    <channel-type id="color_mode">
+        <item-type>Number</item-type>
+        <label>Color</label>
+        <description>The color mode channel allows to control the color mode of a light.
+        </description>
+        <category>ColorLight</category>
+        <state readOnly="true">
+            <options>
+                <option value="0">Hue and Saturation</option>
+                <option value="1">X and Y</option>
+                <option value="2">Color Temperature</option>
```

Comment:
What happens if you send a command to the color channel when this mode is active?

Reply:
I'm not sure what you mean? What "mode" is active? This channel is an indicator of the mode of the bulb - it can't be controlled (maybe I didn't make it read-only. If not, I'll update this).

The way ZigBee bulbs work is that bulbs will switch this mode indicator based on the mode they are in, which in turn is based on the commands they are sent.